modules/router/templates/assets_origin.conf.erb

```diff
@@ -121,4 +121,8 @@ server {
     proxy_pass <%= @upstream_ssl ? 'https' : 'http' %>://static.<%= @app_domain %>;
     <%- end %>
   }
+
+  location /googlec908b3bc32386239.html {
+    alias /var/www/googlec908b3bc32386239.html;
```

Comment:
I think the `return` [1] statement could be used here, so something like:

```
  return 200 'google-site-verification: googlec908b3bc32386239.html';
```

1: http://nginx.org/en/docs/http/ngx_http_rewrite_module.html#return

That would remove the need to have the separate file, as well as the related configuration.

Reply:
That's neat, will change!